SU-SWS/stanford_profile
# D8CORE-336 Imagemagick module

# READY FOR REVIEW

# Summary
- Enable and configure imagemagick

# Needed By (Date)
- 8/30

# Urgency
- low

# Steps to Test
1. use this branch `composer require su-sws/stanford_profile:D8CORE-336-imagemagick`
1. `drush cim -y`
1. add a new image to the site
1. ensure image styles are still generated

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
